lib/drivers/medtronic/medtronicDriver.js

```diff
@@ -1586,11 +1586,10 @@ module.exports = function (config) {
     cleanup: function (progress, data, cb) {
       debug('in cleanup');
 
-      /* TODO: put this back in when we have new sound/earcon
       if(isBrowser) {
-        var snd = new Audio('../../../sounds/underwater.wav');
+        var snd = new Audio('../../../sounds/UploadSucceeded.wav');
```

Comment:
Ok, cool. I was a little confused about it since I couldn't really nail down a place to put the failure case. I'll take out the audio piece from the medtronic specific case.